nav2_collision_monitor/include/nav2_collision_monitor/polygon.hpp

```diff
@@ -251,6 +264,10 @@ class Polygon
   ActionType action_type_;
   /// @brief Minimum number of data readings within a zone to trigger the action
   int min_points_;
+  /// @brief Minimum number of consecutive data with minimum min_points to trigger the action
+  int min_consecutive_in_;
+  /// @brief Minimum number of consecutive data without minimum min_points to stop the action
+  int min_consecutive_out_;
```

Comment:
Thoughts about a duration rather than a number of consecutive cycles? I would think that having the system blocked for > 0.5 seconds or something is more intuitive than 5 cycles, or what have you. I think that would be more intuitive to me, but open to leaving as is if others prefer this (@tonynajjar ?)

Reply:
I agree that specifying a duration would be more intuitive especially because the CM and CD's run on a rate different than the rate of the measurements. For CD it's a configurable rate and for CM it's the rate of the cmd_vel. The "meaning" of `min_consecutive_out_` is dependent on the difference between the CM/CD rate and the measurement rate, if that makes sense.

TLDR: I would vote for duration

Reply:
@gennartan what do you think?

Reply:
Specifying a duration is a better idea, I will update the PR soon and come back with better names for the variables